### PR TITLE
Fixed #121 Add toolbar and toolbar back button on explorer

### LINKED ISSUE #121: Add toolbar and toolbar back button on explorer
Toolbar is a full screen and there is not way (except by using the back hardware button) to go back to the main screen

### PR DESCRIPTION
**Pull request by pg301**
@garg-anuj and @danielpassos 
screenshot_1488997404
I have added toolbar and toolbar back button on explorer. I have used the existing toolbar layout and added that in the InventoryExplorerActivity. Please review it and let me know if some more fixes are required.

### CODE REVIEW

**Comment by KeenWarrior** [comment]
@pg301 Run `mvn license:format` for correcting license headers

`mvn clean install` to check if license and checkstyle are satisfied

**Comment by pg301** [comment]
@garg-anuj Yeah! I am doing that now. Will keep you posted.

File: mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java [hunk 1]
```diff
@@ -65,14 +68,18 @@
     private TreeNode.BaseNodeViewHolder holder;
     private TreeNode root;
     private Callback<String> callback;
+    Toolbar toolbar;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_inventory_explorer);
+        toolbar = (Toolbar) findViewById(R.id.toolbar);
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java]: @pg301 Hawkular Android Client use ButterKnife for binding UI. Mostly that means just @BindView(R.id.id_of_item) over UI instance and 

`ButterKnife.bind(this, getView()); 
`
Can you please visit  [sample](https://github.com/hawkular/hawkular-android-client/blob/master/mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java) to have clearer understanding

Happy Open Source

Comment (pg301) [reply mobile/src/main/java/org/hawkular/client/android/explorer/InventoryExplorerActivity.java]: @garg-anuj I have now changed the implementation of this. I am using butterknife binding 

 @BindView(R.id.toolbar)
    Toolbar toolbar;
Will update this PR soon.

**Comment by danielpassos** [comment]
👍

**Comment by danielpassos** [comment]
Awesome

**Comment by danielpassos** [comment]
Landed at https://github.com/hawkular/hawkular-android-client/commit/04506be571310b5923f64eac95674ac489c3eb77

**Comment by pg301** [comment]
Thank you @danielpassos  :)

**Pull request closed without merging**